app/services/idv/steps/inherited_proofing/agreement_step.rb

```diff
@@ -14,6 +16,19 @@ def form_submit
         def consent_form_params
           params.require(:inherited_proofing).permit(:ial2_consent_given)
         end
+
+        def enqueue_job
+          doc_capture_session = create_document_capture_session(
+            inherited_proofing_verify_step_document_capture_session_uuid_key,
+          )
+
+          doc_capture_session.create_doc_auth_session
+
+          InheritedProofingJob.perform_later(
+            controller.inherited_proofing_service_provider_id,
+            controller.inherited_proofing_service_provider_data, doc_capture_session.uuid
```

Comment:
Just formatting, but I'd break this line across 2 lines. No biggie, take or leave :)